base/error.jl

```diff
@@ -50,61 +50,43 @@ macro assert(ex, msgs...)
     :($(esc(ex)) ? $(nothing) : throw(Main.Base.AssertionError($msg)))
 end
 
+const DEFAULT_RETRY_N = 1
+const DEFAULT_RETRY_ON = e->true
+const DEFAULT_RETRY_MAX_DELAY = 10.0
 
 """
-    retry(f, [condition]; n=3; max_delay=10) -> Function
+    retry(f, [retry_on]; n=DEFAULT_RETRY_N, max_delay=DEFAULT_RETRY_MAX_DELAY) -> Function
```

Comment:
Should `DEFAULT_RETRY_N` be used in the documentation? It is not exported or documented itself.

In fact, excluding the documentation strings, `DEFAULT_RETRY_N` is only used in one place. It should probably just be removed.